packages/openapi-generator/src/service-mapping.ts

```diff
@@ -0,0 +1,23 @@
+import { readJSON } from '@sap-cloud-sdk/util';
+import { GeneratorOptions } from './options';
+export interface VdmMapping {
+  [fileName: string]: ServiceMapping;
+}
+
+export interface ServiceMapping {
+  directoryName: string;
+  npmPackageName: string;
+}
+
+/**
+ * Read the service mapping from a given path provided in the options.
+ * @param options The options provided from the cli command.
+ * @returns The parsed service mapping object.
+ */
+export function readServiceMapping(options: GeneratorOptions): VdmMapping {
+  return (
+    (options.serviceMapping &&
```

Comment:
I find this hard to read, also because I am not used to return (A && B). I would assume that the method return true or false but it obviously must return the readJson result or {}

Reply:
when the option is provided, returns the content of the file, otherwise, `{}` is the default value.